packages/@react-aria/focus/src/FocusScope.tsx

```diff
@@ -184,6 +184,19 @@ function getScopeRoot(scope: HTMLElement[]) {
 function useFocusContainment(scopeRef: RefObject<HTMLElement[]>, contain: boolean) {
   let focusedNode = useRef<HTMLElement>();
 
+  // Loop through the scopeRef array, and check if scopeRef contains
+  // a FocusScope element (determined by two span children). If it does,
+  // pop it.
+  scopeRef.current.forEach(element => {
+    // How do I check for a FocusScope element? Maybe check if it
+    // has two spans. This might not be the best solution for future
+    // development
+    var arr = [...element.children]
+    arr.forEach((item, index, object) => {
```

Comment:
Unfortunately I worry that it might not be enough to just modify `scopeRef`. What if `FocusScope` is not a direct child, but instead is nested deeply inside the child structure? I think we should modify `isElementInScope` check, to filter out events that happen inside another `FocusScope`